pages/account/manage/testimonials.js

```diff
@@ -99,75 +100,78 @@ export default function ManageTestimonials({ BASE_URL, testimonials }) {
           additionalMessage="Your profile information has been saved successfully."
         />
 
-        <div>
-          <h3 className="text-lg font-medium leading-6 text-primary-high mb-4">
-            Testimonials you have received, toggle to show on your Profile
-          </h3>
-        </div>
+        {!testimonials?.length ? <Alert type="info" message="No testimonials found" /> :
+          <>
```

Comment:
I think the `:` makes it difficult to read, other places in the code we use the condition twice

- `!testimonials ...`
- `testimonials ...`

Reply:
Okay. I will get on it now.

Reply:
fixed 

![testimonial fix](https://github.com/EddieHubCommunity/LinkFree/assets/110388816/1dadc9b3-1b39-4180-9cee-bc480bb8da87)
![fix](https://github.com/EddieHubCommunity/LinkFree/assets/110388816/024b2005-eac0-4083-b277-42b137769592)

Reply:
I made a new update
![test update](https://github.com/EddieHubCommunity/LinkFree/assets/110388816/9eeabd25-d66a-4a27-aeca-6dd93bed34a6)

result
![change](https://github.com/EddieHubCommunity/LinkFree/assets/110388816/ca54eda9-8744-4239-8072-1af8d334de5b)